ocaml/idl/datamodel.ml

```diff
@@ -641,6 +641,8 @@ let _ =
     ~doc:"VGPU type is not one of the PGPU's supported types." ();
   error Api_errors.vgpu_type_not_compatible_with_running_type ["pgpu"; "type"; "running_type"]
     ~doc:"VGPU type is not compatible with one or more of the VGPU types currently running on this PGPU" ();
+  error Api_errors.vgpu_not_compatible_with_destination_pgpu ["reason"]
```

Comment:
But is it possible to specify a specific PGPU through the API?
Should this instead be something like
`"This VM has a VGPU that is not compatible with any PGPU on the specified destination host."`
?